src/transformers/modeling_utils.py

```diff
@@ -1927,6 +1936,23 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             subfolder (`str`, *optional*, defaults to `""`):
                 In case the relevant files are located inside a subfolder of the model repo on huggingface.co, you can
                 specify the folder name here.
+            variant (`str`, *optional*):
+                If specified load weights with the `variant` suffix. If no weights with the `variant` suffix are
+                available an error is thrown.
+
+                <Tip>
+
+                Passing `variant` is useful to load weights serialized in float16. *E.g.*:
```

Comment:
To do: we should standardize on the names for common variants and document them. In the Conversation tab we used `fp16` instead.